src/app/index.js

```diff
@@ -618,6 +602,8 @@ export default yeoman.generators.Base.extend({
     if (this.options.hurry) {
       return;
     }
+    shelljs.exec('git add --all');
+    shelljs.exec('git commit -a -m "generator run"');
```

Comment:
I'm not sure about this. I don't really like the idea of creating commits implicitly.